src/README.md

```diff
@@ -830,6 +831,15 @@ The following additional attributes are supported:
 
   See also [add_attachments](#add_attachments)
 
+
+* Connected issues are given as a dict or a list of dicts using the key `_issue_data` which contains the attributes for one or more `ogit/Automation/AutomationIssue`. Additional attributes for the issue(s) will be set as needed:
+    * `ogit/_type`: Will be set automatically to `ogit/Automation/AutomationIssue`
+    * `ogit/Automation/originNode`: Will be set to the `ogit/_id` of the vertex that has been created in stage 1).
```

Comment:
- 'ogit/Automation/originNode' might be also different from what has been created in stage 1), I propose to allow set it if needed (same as ogit/subject)
- Additionally, ogit/_scope, ogit/_owner attributes for issue data to handle cross org creation.

Reply:
Okay. This was worded badly. You can set all attributes as you like it, but the three mentioned here are just special cases.
* If you want ogit/Automation/originNode to be something else, you do NOT write the issue in the same json object as the original vertex: You create the issue on its own in a separate object and use the `xid:ogit/Automation/originNode`-Header to reference to another vertex with this xid (or just use `ogit/Automation/originNode` with the id of the other vertex if known).
* `ogit/_scope` and `ogit/_owner` can be set freely as headers `issue:0:ogit/_scope` and `issue:0:ogit/_owner`.

Reply:
See https://github.com/arago/hiro-client-python/pull/31#issuecomment-1017717304